api/http/server.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/portainer/portainer/api/http/handler/endpoints"
 	"github.com/portainer/portainer/api/http/handler/file"
 	"github.com/portainer/portainer/api/http/handler/helm"
+	"github.com/portainer/portainer/api/http/handler/hostmanagement/intel"
```

Comment:
So I think what Sven was getting at was maybe having hostmanagement/openamt and hostmanagement/ipmi etc.
As IPMI is another host management protocol.  I'd still call it openamt though as that is the name of the tech.